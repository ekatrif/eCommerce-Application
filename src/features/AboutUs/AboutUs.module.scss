.aboutUs {
  display: flex;
  flex-direction: column;
  row-gap: 3rem;
  width: 90%;
  margin: 0 auto;
}

.aboutUs__title {
  margin-top: 1rem;
  font-size: 2rem;
  margin-bottom: 30px;
  text-align: center;
}

.aboutUs__subTitle {
  font-size: 1.3rem;
  text-align: center;
  margin-bottom: 20px;
}

.aboutUs__team {
  display: flex;
  justify-content: space-around;
  column-gap: 25px;
}

.aboutUs__contributions {
  width: 100%;
  display: flex;
  flex-direction: column;
  margin: 0 auto;
  padding: 30px;
  max-width: 900px;
  background-color: rgba(173, 173, 173, 0.145);
}

.aboutUs__collaboration {
  max-width: 900px;
  padding: 30px;
  margin: 0 auto;
}

.aboutUs__logo {
  margin: 1rem auto;
}

@media (max-width: 1100px) {
  .aboutUs__team {
    flex-direction: column;
    align-items: center;
    row-gap: 25px;
  }
}