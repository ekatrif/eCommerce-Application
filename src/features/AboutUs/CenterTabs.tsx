/* import * as React from 'react';
import Tabs from '@mui/material/Tabs';
import Tab from '@mui/material/Tab';
import Typography from '@mui/material/Typography';
import Box from '@mui/material/Box';
import Avatar from '@mui/material/Avatar';

interface TabPanelProps {
  children?: React.ReactNode;
  index: number;
  value: number;
}

function TabPanel(props: TabPanelProps): JSX.Element {
  const { children, value, index, ...other } = props;

  return (
    <div
      role="tabpanel"
      hidden={value !== index}
      id={`vertical-tabpanel-${index}`}
      aria-labelledby={`vertical-tab-${index}`}
      {...other}
    >
      {value === index && (
        <Box sx={{ p: 3 }}>
          <Typography>{children}</Typography>
        </Box>
      )}
    </div>
  );
}

interface Ia11yProps {
  id: string;
  'aria-controls': string;
}

function a11yProps(index: number): Ia11yProps {
  return {
    id: `vertical-tab-${index}`,
    'aria-controls': `vertical-tabpanel-${index}`,
  };
}

export default function VerticalTabs(): JSX.Element {
  const [value, setValue] = React.useState(0);

  const handleChange = (event: React.SyntheticEvent, newValue: number): void => {
    setValue(newValue);
  };

  return (
    <Box sx={{ flexGrow: 1, bgcolor: 'background.paper', display: 'flex', height: 224 }}>
      <Tabs
        orientation="vertical"
        variant="scrollable"
        value={value}
        onChange={handleChange}
        aria-label="Vertical tabs example"
        sx={{ borderRight: 1, borderColor: 'divider' }}
      >
        <Tab label={<Avatar alt="Remy Sharp" src="../../assets/images/team/oleg.jpg" />} {...a11yProps(0)} />
        <Tab label={<Avatar alt="Remy Sharp" src="../../assets/images/team/kate.jpg" />} {...a11yProps(1)} />
        <Tab label={<Avatar alt="Remy Sharp" src="../../assets/images/team/andrey.jpg" />} {...a11yProps(2)} />
      </Tabs>
      <TabPanel value={value} index={0}>
        Вклад Олега был в ...
      </TabPanel>
      <TabPanel value={value} index={1}>
        Вклад Кати был в ...
      </TabPanel>
      <TabPanel value={value} index={2}>
        Вклад Андрея был в ...
      </TabPanel>
    </Box>
  );
} */

import * as React from 'react';
import Tabs from '@mui/material/Tabs';
import Tab from '@mui/material/Tab';
import Typography from '@mui/material/Typography';
import Box from '@mui/material/Box';
import Avatar from '@mui/material/Avatar';

interface TabPanelProps {
  children?: React.ReactNode;
  index: number;
  value: number;
}

function CustomTabPanel(props: TabPanelProps): JSX.Element {
  const { children, value, index, ...other } = props;

  return (
    <div
      role="tabpanel"
      hidden={value !== index}
      id={`simple-tabpanel-${index}`}
      aria-labelledby={`simple-tab-${index}`}
      {...other}
    >
      {value === index && (
        <Box sx={{ p: 3 }}>
          <Typography>{children}</Typography>
        </Box>
      )}
    </div>
  );
}

interface Ia11yProps {
  id: string;
  'aria-controls': string;
}

function a11yProps(index: number): Ia11yProps {
  return {
    id: `simple-tab-${index}`,
    'aria-controls': `simple-tabpanel-${index}`,
  };
}

export default function CenterTabs(): JSX.Element {
  const [value, setValue] = React.useState(0);

  const handleChange = (event: React.SyntheticEvent, newValue: number): void => {
    setValue(newValue);
  };

  return (
    <Box sx={{ maxWidth: '850px' }}>
      <Box sx={{ borderBottom: 1, borderColor: 'divider' }}>
        <Tabs value={value} onChange={handleChange} aria-label="basic tabs example" centered>
          <Tab label={<Avatar alt="Remy Sharp" src="../../assets/images/team/oleg.jpg" />} {...a11yProps(0)} />
          <Tab label={<Avatar alt="Remy Sharp" src="../../assets/images/team/kate.jpg" />} {...a11yProps(1)} />
          <Tab label={<Avatar alt="Remy Sharp" src="../../assets/images/team/andrey.jpg" />} {...a11yProps(2)} />
        </Tabs>
      </Box>
      <CustomTabPanel value={value} index={0}>
        Вклад Олега был в ... Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita, qui autem! Rem atque at
        adipisci modi. Tempore vitae laborum cupiditate, quidem placeat obcaecati excepturi blanditiis molestiae
        dignissimos explicabo totam fuga. Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita, qui autem!
        Rem atque a adipisci modi. Tempore vitae laborum cupiditate, quidem placeat obcaecati excepturi blanditiis
        molestiae dignissimos explicabo totam fuga. quidem placeat obcaecati excepturi blanditiis molestiae dignissimos
        explicabo totam fuga. Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita, qui autem! Rem atque a
        adipisci modi. Tempore vitae laborum cupiditate, quidem placeat obcaecati excepturi blanditiis molestiae
        dignissimos explicabo.
      </CustomTabPanel>
      <CustomTabPanel value={value} index={1}>
        Вклад Кати был в ... Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita, qui autem! Rem atque at
        adipisci modi. Tempore vitae laborum cupiditate, quidem placeat obcaecati excepturi blanditiis molestiae
        dignissimos explicabo totam fuga. Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita, qui autem!
        Rem atque a adipisci modi. Tempore vitae laborum cupiditate, quidem placeat obcaecati excepturi blanditiis
        molestiae dignissimos explicabo totam fuga. quidem placeat obcaecati excepturi blanditiis molestiae dignissimos
        explicabo totam fuga. Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita, qui autem! Rem atque a
        adipisci modi. Tempore vitae laborum cupiditate, quidem placeat obcaecati excepturi blanditiis molestiae
        dignissimos explicabo.
      </CustomTabPanel>
      <CustomTabPanel value={value} index={2}>
        Вклад Андрея был в ... Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita, qui autem! Rem atque
        adipisci modi. Tempore vitae laborum cupiditate, quidem placeat obcaecati excepturi blanditiis molestiae
        dignissimos explicabo totam fuga. Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita, qui autem!
        Rem atque a adipisci modi. Tempore vitae laborum cupiditate, quidem placeat obcaecati excepturi blanditiis
        molestiae dignissimos explicabo totam fuga. quidem placeat obcaecati excepturi blanditiis molestiae dignissimos
        explicabo totam fuga. Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita, qui autem! Rem atque a
        adipisci modi. Tempore vitae laborum cupiditate, quidem placeat obcaecati excepturi blanditiis molestiae
        dignissimos explicabo.
      </CustomTabPanel>
    </Box>
  );
}
