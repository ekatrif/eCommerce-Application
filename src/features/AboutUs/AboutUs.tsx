import React from 'react';
import classes from './AboutUs.module.scss';
import UserCard from '../../components/UI/userCard/UserCard';
import CenterTabs from './CenterTabs';
import FolderList from './FolderList';
import team from '../../assets/data/team';

const AboutUs: React.FC = () => {
  return (
    <div className={classes.aboutUs}>
      <div>
        <h2 className={classes.aboutUs__title}>Team Members</h2>
        <div className={classes.aboutUs__team}>
          {team.map((person, index) => (
            <UserCard
              key={index}
              name={person.name}
              image={person.image}
              role={person.role}
              git={person.git}
              bio={person.bio}
            />
          ))}
        </div>
      </div>
      <div className={classes.aboutUs__contributions}>
        <h2 className={classes.aboutUs__title}>Contributions</h2>
        <CenterTabs />
      </div>
      <div className={classes.aboutUs__collaboration}>
        <h2 className={classes.aboutUs__title}>Collaboration</h2>
        <p className={classes.aboutUs__subTitle}>The success of our team depended on:</p>
        <FolderList />
      </div>
      <a className={classes.aboutUs__logo} href="https://rs.school/js/">
        <img src="/assets/svg/rs_school_js.svg" alt="RS_School_logo" width="100%"></img>
      </a>
    </div>
  );
};

export default AboutUs;
