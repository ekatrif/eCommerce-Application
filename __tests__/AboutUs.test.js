import React from 'react';
import renderer from 'react-test-renderer';
import AboutUs from '../src/features/AboutUs/AboutUs'

test('AboutUs renders correctly', () => {
  const tree = renderer
    .create(<AboutUs />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
