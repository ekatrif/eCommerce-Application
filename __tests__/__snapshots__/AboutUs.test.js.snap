// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AboutUs renders correctly 1`] = `
<div
  className="aboutUs"
>
  <div>
    <h2
      className="aboutUs__title"
    >
      Team Members
    </h2>
    <div
      className="aboutUs__team"
    >
      <div
        className="userCard"
      >
        <div>
          <img
            className="userCard__image"
            src="../assets/images/team/oleg.jpg"
          />
          <h2
            className="userCard__name"
          >
            Oleg Polovinko
          </h2>
          <p
            className="userCard__role"
          >
            Team Lead, Frontend Developer
          </p>
        </div>
        <div
          className="userCard__text"
        >
          <ul />
          <a
            className="userCard__git"
            href="https://github.com/sheritsh"
            target="_blank"
          >
            sheritsh
          </a>
        </div>
      </div>
      <div
        className="userCard"
      >
        <div>
          <img
            className="userCard__image"
            src="../assets/images/team/kate.jpg"
          />
          <h2
            className="userCard__name"
          >
            Ekaterina Trifonova
          </h2>
          <p
            className="userCard__role"
          >
            Frontend Developer
          </p>
        </div>
        <div
          className="userCard__text"
        >
          <ul>
            <li>
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="KeyboardArrowRightIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.59 16.59 13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                />
              </svg>
              Graduated from Bauman Moscow State Technical University in 2017.
            </li>
            <li>
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="KeyboardArrowRightIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.59 16.59 13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                />
              </svg>
              Worked as a technical writer.
            </li>
            <li>
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="KeyboardArrowRightIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.59 16.59 13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                />
              </svg>
              Developed websites on CMS Wordpress, Prestashop, Modx, Tilda etc.
            </li>
            <li>
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="KeyboardArrowRightIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.59 16.59 13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                />
              </svg>
              Completed a front-end development course at the "Professional" training center in 2021.
            </li>
            <li>
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="KeyboardArrowRightIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.59 16.59 13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                />
              </svg>
              Started learning frontend development on her own.
            </li>
            <li>
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="KeyboardArrowRightIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.59 16.59 13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                />
              </svg>
              Enrolled in stage 1 in RS School and failed it in 2022.
            </li>
            <li>
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="KeyboardArrowRightIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.59 16.59 13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                />
              </svg>
              Applied for stage 0 with a new cohort.
            </li>
            <li>
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="KeyboardArrowRightIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.59 16.59 13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                />
              </svg>
              Finished stage 1 and moved on to the second in 2023.
            </li>
            <li>
              <svg
                aria-hidden={true}
                className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="KeyboardArrowRightIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.59 16.59 13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                />
              </svg>
              Currently looking for a job as a front-end developer or internship.
            </li>
          </ul>
          <a
            className="userCard__git"
            href="https://github.com/ekatrif"
            target="_blank"
          >
            ekatrif
          </a>
        </div>
      </div>
      <div
        className="userCard"
      >
        <div>
          <img
            className="userCard__image"
            src="../assets/images/team/andrey.jpg"
          />
          <h2
            className="userCard__name"
          >
            Andrey Nezhdanov
          </h2>
          <p
            className="userCard__role"
          >
            Frontend Developer
          </p>
        </div>
        <div
          className="userCard__text"
        >
          <ul />
          <a
            className="userCard__git"
            href="https://github.com/montek1o"
            target="_blank"
          >
            montek1o
          </a>
        </div>
      </div>
    </div>
  </div>
  <div
    className="aboutUs__contributions"
  >
    <h2
      className="aboutUs__title"
    >
      Contributions
    </h2>
    <div
      className="MuiBox-root css-pzaa1o"
    >
      <div
        className="MuiBox-root css-1gsv261"
      >
        <div
          className="MuiTabs-root css-1ujnqem-MuiTabs-root"
        >
          <div
            className="MuiTabs-scroller MuiTabs-fixed css-jpln7h-MuiTabs-scroller"
            style={
              {
                "marginBottom": -0,
                "overflow": "hidden",
              }
            }
          >
            <div
              aria-label="basic tabs example"
              aria-orientation={null}
              className="MuiTabs-flexContainer MuiTabs-centered css-1wf8b0h-MuiTabs-flexContainer"
              onKeyDown={[Function]}
              role="tablist"
            >
              <button
                aria-controls="simple-tabpanel-0"
                aria-selected={true}
                className="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary Mui-selected css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
                disabled={false}
                id="simple-tab-0"
                onBlur={[Function]}
                onClick={[Function]}
                onContextMenu={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="tab"
                tabIndex={0}
                type="button"
              >
                <div
                  className="MuiAvatar-root MuiAvatar-circular css-1wlk0hk-MuiAvatar-root"
                >
                  <img
                    alt="Remy Sharp"
                    className="MuiAvatar-img css-1pqm26d-MuiAvatar-img"
                    src="../../assets/images/team/oleg.jpg"
                  />
                </div>
                <span
                  className="MuiTabs-indicator css-1aquho2-MuiTabs-indicator"
                  style={{}}
                />
              </button>
              <button
                aria-controls="simple-tabpanel-1"
                aria-selected={false}
                className="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
                disabled={false}
                id="simple-tab-1"
                onBlur={[Function]}
                onClick={[Function]}
                onContextMenu={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="tab"
                tabIndex={-1}
                type="button"
              >
                <div
                  className="MuiAvatar-root MuiAvatar-circular css-1wlk0hk-MuiAvatar-root"
                >
                  <img
                    alt="Remy Sharp"
                    className="MuiAvatar-img css-1pqm26d-MuiAvatar-img"
                    src="../../assets/images/team/kate.jpg"
                  />
                </div>
              </button>
              <button
                aria-controls="simple-tabpanel-2"
                aria-selected={false}
                className="MuiButtonBase-root MuiTab-root MuiTab-textColorPrimary css-1h9z7r5-MuiButtonBase-root-MuiTab-root"
                disabled={false}
                id="simple-tab-2"
                onBlur={[Function]}
                onClick={[Function]}
                onContextMenu={[Function]}
                onDragLeave={[Function]}
                onFocus={[Function]}
                onKeyDown={[Function]}
                onKeyUp={[Function]}
                onMouseDown={[Function]}
                onMouseLeave={[Function]}
                onMouseUp={[Function]}
                onTouchEnd={[Function]}
                onTouchMove={[Function]}
                onTouchStart={[Function]}
                role="tab"
                tabIndex={-1}
                type="button"
              >
                <div
                  className="MuiAvatar-root MuiAvatar-circular css-1wlk0hk-MuiAvatar-root"
                >
                  <img
                    alt="Remy Sharp"
                    className="MuiAvatar-img css-1pqm26d-MuiAvatar-img"
                    src="../../assets/images/team/andrey.jpg"
                  />
                </div>
              </button>
            </div>
          </div>
        </div>
      </div>
      <div
        aria-labelledby="simple-tab-0"
        hidden={false}
        id="simple-tabpanel-0"
        role="tabpanel"
      >
        <div
          className="MuiBox-root css-19kzrtu"
        >
          <p
            className="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
          >
            Вклад Олега был в ... Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita, qui autem! Rem atque at adipisci modi. Tempore vitae laborum cupiditate, quidem placeat obcaecati excepturi blanditiis molestiae dignissimos explicabo totam fuga. Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita, qui autem! Rem atque a adipisci modi. Tempore vitae laborum cupiditate, quidem placeat obcaecati excepturi blanditiis molestiae dignissimos explicabo totam fuga. quidem placeat obcaecati excepturi blanditiis molestiae dignissimos explicabo totam fuga. Lorem ipsum dolor sit amet consectetur adipisicing elit. Expedita, qui autem! Rem atque a adipisci modi. Tempore vitae laborum cupiditate, quidem placeat obcaecati excepturi blanditiis molestiae dignissimos explicabo.
          </p>
        </div>
      </div>
      <div
        aria-labelledby="simple-tab-1"
        hidden={true}
        id="simple-tabpanel-1"
        role="tabpanel"
      />
      <div
        aria-labelledby="simple-tab-2"
        hidden={true}
        id="simple-tabpanel-2"
        role="tabpanel"
      />
    </div>
  </div>
  <div
    className="aboutUs__collaboration"
  >
    <h2
      className="aboutUs__title"
    >
      Collaboration
    </h2>
    <p
      className="aboutUs__subTitle"
    >
      The success of our team depended on:
    </p>
    <ul
      className="MuiList-root MuiList-padding css-1cq5zt0-MuiList-root"
    >
      <li
        className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-ovffbk-MuiListItem-root"
        disabled={false}
      >
        <div
          className="MuiListItemAvatar-root css-1e9lk82-MuiListItemAvatar-root"
        >
          <div
            className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-2s90m6-MuiAvatar-root"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="PermPhoneMsgIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M20 15.5c-1.25 0-2.45-.2-3.57-.57-.35-.11-.74-.03-1.02.24l-2.2 2.2c-2.83-1.44-5.15-3.75-6.59-6.58l2.2-2.21c.28-.27.36-.66.25-1.01C8.7 6.45 8.5 5.25 8.5 4c0-.55-.45-1-1-1H4c-.55 0-1 .45-1 1 0 9.39 7.61 17 17 17 .55 0 1-.45 1-1v-3.5c0-.55-.45-1-1-1zM12 3v10l3-3h6V3h-9z"
              />
            </svg>
          </div>
        </div>
        <div
          className="MuiListItemText-root css-tlelie-MuiListItemText-root"
        >
          <span
            className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
          >
            Weekly calls to discord
          </span>
        </div>
      </li>
      <li
        className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-ovffbk-MuiListItem-root"
        disabled={false}
      >
        <div
          className="MuiListItemAvatar-root css-1e9lk82-MuiListItemAvatar-root"
        >
          <div
            className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-2s90m6-MuiAvatar-root"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="EditNoteIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M3 10h11v2H3v-2zm0-2h11V6H3v2zm0 8h7v-2H3v2zm15.01-3.13.71-.71c.39-.39 1.02-.39 1.41 0l.71.71c.39.39.39 1.02 0 1.41l-.71.71-2.12-2.12zm-.71.71-5.3 5.3V21h2.12l5.3-5.3-2.12-2.12z"
              />
            </svg>
          </div>
        </div>
        <div
          className="MuiListItemText-root css-tlelie-MuiListItemText-root"
        >
          <span
            className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
          >
            Daily chat reports on the progress of tasks
          </span>
        </div>
      </li>
      <li
        className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-ovffbk-MuiListItem-root"
        disabled={false}
      >
        <div
          className="MuiListItemAvatar-root css-1e9lk82-MuiListItemAvatar-root"
        >
          <div
            className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-2s90m6-MuiAvatar-root"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="TelegramIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M9.78 18.65l.28-4.23 7.68-6.92c.34-.31-.07-.46-.52-.19L7.74 13.3 3.64 12c-.88-.25-.89-.86.2-1.3l15.97-6.16c.73-.33 1.43.18 1.15 1.3l-2.72 12.81c-.19.91-.74 1.13-1.5.71L12.6 16.3l-1.99 1.93c-.23.23-.42.42-.83.42z"
              />
            </svg>
          </div>
        </div>
        <div
          className="MuiListItemText-root css-tlelie-MuiListItemText-root"
        >
          <span
            className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
          >
            Constant communication in telegram
          </span>
        </div>
      </li>
      <li
        className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-ovffbk-MuiListItem-root"
        disabled={false}
      >
        <div
          className="MuiListItemAvatar-root css-1e9lk82-MuiListItemAvatar-root"
        >
          <div
            className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-2s90m6-MuiAvatar-root"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="AddTaskIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M22 5.18 10.59 16.6l-4.24-4.24 1.41-1.41 2.83 2.83 10-10L22 5.18zM12 20c-4.41 0-8-3.59-8-8s3.59-8 8-8c1.57 0 3.04.46 4.28 1.25l1.45-1.45C16.1 2.67 14.13 2 12 2 6.48 2 2 6.48 2 12s4.48 10 10 10c1.73 0 3.36-.44 4.78-1.22l-1.5-1.5c-1 .46-2.11.72-3.28.72zm7-5h-3v2h3v3h2v-3h3v-2h-3v-3h-2v3z"
              />
            </svg>
          </div>
        </div>
        <div
          className="MuiListItemText-root css-tlelie-MuiListItemText-root"
        >
          <span
            className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
          >
            Using the task board in Trello
          </span>
        </div>
      </li>
      <li
        className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-ovffbk-MuiListItem-root"
        disabled={false}
      >
        <div
          className="MuiListItemAvatar-root css-1e9lk82-MuiListItemAvatar-root"
        >
          <div
            className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-2s90m6-MuiAvatar-root"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="Diversity3Icon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M6.32 13.01c.96.02 1.85.5 2.45 1.34C9.5 15.38 10.71 16 12 16s2.5-.62 3.23-1.66c.6-.84 1.49-1.32 2.45-1.34-.72-1.22-3.6-2-5.68-2-2.07 0-4.96.78-5.68 2.01zM4 13c1.66 0 3-1.34 3-3S5.66 7 4 7s-3 1.34-3 3 1.34 3 3 3zm16 0c1.66 0 3-1.34 3-3s-1.34-3-3-3-3 1.34-3 3 1.34 3 3 3zm-8-3c1.66 0 3-1.34 3-3s-1.34-3-3-3-3 1.34-3 3 1.34 3 3 3z"
              />
              <path
                d="M21 14h-3.27c-.77 0-1.35.45-1.68.92-.04.06-1.36 2.08-4.05 2.08-1.43 0-3.03-.64-4.05-2.08-.39-.55-1-.92-1.68-.92H3c-1.1 0-2 .9-2 2v4h7v-2.26c1.15.8 2.54 1.26 4 1.26s2.85-.46 4-1.26V20h7v-4c0-1.1-.9-2-2-2z"
              />
            </svg>
          </div>
        </div>
        <div
          className="MuiListItemText-root css-tlelie-MuiListItemText-root"
        >
          <span
            className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
          >
            Regular code review by mentor
          </span>
        </div>
      </li>
      <li
        className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-ovffbk-MuiListItem-root"
        disabled={false}
      >
        <div
          className="MuiListItemAvatar-root css-1e9lk82-MuiListItemAvatar-root"
        >
          <div
            className="MuiAvatar-root MuiAvatar-circular MuiAvatar-colorDefault css-2s90m6-MuiAvatar-root"
          >
            <svg
              aria-hidden={true}
              className="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="CelebrationIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="m2 22 14-5-9-9zm12.53-9.47 5.59-5.59c.49-.49 1.28-.49 1.77 0l.59.59 1.06-1.06-.59-.59c-1.07-1.07-2.82-1.07-3.89 0l-5.59 5.59 1.06 1.06zm-4.47-5.65-.59.59 1.06 1.06.59-.59c1.07-1.07 1.07-2.82 0-3.89l-.59-.59-1.06 1.07.59.59c.48.48.48 1.28 0 1.76zm7 5-1.59 1.59 1.06 1.06 1.59-1.59c.49-.49 1.28-.49 1.77 0l1.61 1.61 1.06-1.06-1.61-1.61c-1.08-1.07-2.82-1.07-3.89 0zm-2-6-3.59 3.59 1.06 1.06 3.59-3.59c1.07-1.07 1.07-2.82 0-3.89l-1.59-1.59-1.06 1.06 1.59 1.59c.48.49.48 1.29 0 1.77z"
              />
            </svg>
          </div>
        </div>
        <div
          className="MuiListItemText-root css-tlelie-MuiListItemText-root"
        >
          <span
            className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
          >
            Gifs in pull requests
          </span>
        </div>
      </li>
    </ul>
  </div>
  <a
    className="aboutUs__logo"
    href="https://rs.school/js/"
  >
    <img
      alt="RS_School_logo"
      src="/assets/svg/rs_school_js.svg"
      width="100%"
    />
  </a>
</div>
`;
